### PR TITLE
Create Context Menu Item.

### PR DESCRIPTION
**Pull request by theseal**
Create Context Menu Item for easier access to the extension
without having a visible button.

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 var buttons = require('sdk/ui/button/action');
+var cm = require("sdk/context-menu");
```
Comment (Tenzer) [inline lib/main.js]: Could you replace this with single-quotes please, along with the other strings you added?

File: lib/main.js [hunk 2]
```diff
@@ -76,6 +77,13 @@ var disk_visitor = {
     }
 };
 
+cm.Item({
+    label: 'Clear Site Cache',
+    context: cm.URLContext("*"),
+    contentScript: 'self.on("click", self.postMessage);',
+    onMessage: function () { clearCache () }
```
Comment (Tenzer) [inline lib/main.js]: Couldn't this just be `onMessage: clearCache` like on line 22?

Comment (theseal) [reply lib/main.js]: Hmm. I thought I tried that without success. But now when testing now  works just fine.
Fixed!

File: lib/main.js (hunk 1 above)
Comment (theseal) [reply lib/main.js]: Fixed! 👍

**Comment by theseal** [comment]
Both comment fixed. Let me know if there something else that I have missed.

**Comment by Tenzer** [comment]
Great, thanks for the contributions! I'll see if I can get the setup done on my current laptop so I can publish the changes to the addons site.

**Pull request merged**